### PR TITLE
feat: de output

### PR DESCRIPTION
**Pull request by yeising**
Added volcano plot to visualize differentially expressed genes.
Added CSV of differentially expressed genes under alpha threshold with their normalized read count per sample, log2foldchange and pvalue.

### CODE REVIEW

File: workflow/scripts/de_analysis.py [hunk 1]
```diff
@@ -127,7 +128,10 @@
 # delete rows, which do not meet our p-value criterion
 # the comparison operator is >= because we drop all values >= our desired alpha
 normalized.drop(normalized[cutoff[1] >= snakemake.config["alpha"]].index, inplace=True)
-# through away these columns
+# copy normalized to get output matrix of genes with lfc and pvalue
+de_genes = normalized.copy()
```
Comment (cmeesters) [inline workflow/scripts/de_analysis.py:132]: Why do we need this copy again? And write it out to a file?

In the end, we fall back to the previous data-frame, anyways.

**Comment by yeising** [comment]
At this point, branch has been rebased with #84.

File: workflow/scripts/de_analysis.py (hunk 1 above)
Comment (cmeesters) [reply workflow/scripts/de_analysis.py]: resolved by https://github.com/snakemake-workflows/transcriptome-differential-expression/pull/83/commits/bddf59f81f562588fc963ab7440e4a73ff4e0822

**Review by cmeesters — APPROVED** [review]
nice!

**Pull request merged**